### PR TITLE
Trying to resize image results in zero size image

### PR DESCRIPTION
**Pull request by mightyiam**
Use `jQuery.fn.<height|width>()` for getting image size instead of `jQuery.fn.attr('<height|width>')`.

### CODE REVIEW

File: src/wymeditor/editor/image-handler.js [hunk 1]
```diff
@@ -519,8 +519,8 @@ WYMeditor.ImageHandler.prototype._resizeImage = function (currentMouseY) {
 
     if (!dimensionsRatio) {
         // in order to prevent dimensions ratio corruption
-        var originalHeight = $img.attr('height');
-        var originalWidth = $img.attr('width');
+        var originalHeight = $img.height();
+        var originalWidth = $img.width();
```
Comment (caffodian) [inline src/wymeditor/editor/image-handler.js]: ah, actual height vs. defined height on img?

**Comment by caffodian** [comment]
:+1:

File: src/wymeditor/editor/image-handler.js (hunk 1 above)
Comment (mightyiam) [inline src/wymeditor/editor/image-handler.js]: Yes. I should have been surprised that it actually worked.

Initially, not all images have `height` and `width` attributes. But it did work with those images, nonetheless.

Inside PolicyStat it doesn't, though. So perhaps it is a jQuery difference.

**Comment by mightyiam** [comment]
Thank you

File: src/wymeditor/editor/image-handler.js (hunk 1 above)
Comment (mightyiam) [inline src/wymeditor/editor/image-handler.js]: I should have explained this before you made your review.

**Pull request merged**